docs/traceflow-guide.md

```diff
@@ -1,10 +1,16 @@
 # Traceflow User Guide
 
-Antrea supports using Traceflow for network diagnosis: it generates tracing requests for traffic going through
-Antrea-managed Pod network. Creating a new Traceflow CRD triggers the Traceflow module to inject packet into OVS,
-provide various observation points along the packet's path and populate these observations into the status field of
-the Traceflow CRD. Users can start a new trace simply from kubectl, antctl or Antrea-Octant-Plugin and view Traceflow
-result via CRD, antctl or UI graph.
+Antrea supports using Traceflow for network diagnosis. It can inject a packet
+into OVS on a Node and trace the forwarding patch of the packet across Nodes,
+and it can also trace a matched packet of real traffic from or to a Pod. In
+either case, a Traceflow operation is triggered by a Traceflow CRD which
```

Comment:
I know this is not introduced by this PR but I have noticed we capitalize "Traceflow" in this document but not in the antctl document

Reply:
What is your suggestion? I can change to "Traceflow" for all.

Reply:
Updated antctl.md to use "Traceflow".